### ISSUE TITLE
[Bug]: Calling client config when it is not yet available

### ISSUE THREAD

**Issue by SettingDust** [Fuzss/sneakycurses#14]
### Mod Loader (Required)

Forge

### Minecraft Version (Required)

1.18.2

### Mod Version (Required)

3.3.5

### Steps to Reproduce the Bug (Required)

apotheosis
sneakycurses

```
[19:37:24] [Render thread/ERROR] (Puzzles Lib) Calling client config when it is not yet available! This is a bug! Current loading stage: MISSING_DATA
java.lang.Exception: Client config not yet available
	at TRANSFORMER/puzzleslib@3.3.5/fuzs.puzzleslib.config.ConfigHolderImpl.client(ConfigHolderImpl.java:263)
	at TRANSFORMER/sneakycurses@3.0.0/fuzs.sneakycurses.util.CurseMatcher.isAffected(CurseMatcher.java:26)
	at TRANSFORMER/minecraft@1.18.2/net.minecraft.world.item.Item.handler$cdb000$isFoil$head(Item.java:2516)
	at TRANSFORMER/minecraft@1.18.2/net.minecraft.world.item.Item.m_5812_(Item.java)
	at TRANSFORMER/minecraft@1.18.2/net.minecraft.world.item.ItemStack.m_41790_(ItemStack.java:823)
	at TRANSFORMER/apotheosis@5.3.5/shadows.apotheosis.ench.anvil.ApothAnvilBlock.m_5871_(ApothAnvilBlock.java:108)
```

### Expected Behavior (Required)

no

### Actual Behavior (Required)

error

**Comment by Fuzss** [comment]
Thanks, that's not a big problem, shouldn't cause any issues. I'll fix it in the next version.

**Comment by Fuzss** [comment]
Fixed in v8.0.0.
